src/expressions.js

```diff
@@ -38,17 +38,28 @@ function parse(exp) {
  * @returns {Object} The evaluated parameters to be seeded.
  */
 export function expressionEvaluator(data) {
```

Comment:
Let's remove the `export` so it's clear how this file is used.

Reply:
mmmm, its used in a unit test. suggestions?